plugins/scaffolder/api-report.md

```diff
@@ -317,4 +327,9 @@ export const TextValuePicker: ({
   idSchema,
   placeholder,
 }: FieldProps<string>) => JSX.Element;
+
+// @public
+export const useSecretsContext: () => {
```

Comment:
Wonder if we should make this slightly more verbose, and probably fine to skip the context suffix? Thinking something like

`useTemplateSecrets`
`useFieldExtensionSecrets`
`useTemplateFormSecrets`

Reply:
`useTemplateSecrets` is pretty nice, they're not just confined to `FieldExtensions` as the router will automatically take the `backstageToken` and make that a secret so I think something around `TemplateSecrets` makes the most sense.